openapi-v1.yaml

```diff
@@ -1520,6 +1553,8 @@ definitions:
     description: File types represented as TileDB arrays
     type: string
     enum:
+      # Array
+      - array
```

Comment:
This one scares me since it's a drastic change.
`array` and `group` file types are not supported by REST and we use this enum for filtering in multiple places.

Is it safe to do this change?
Should we create a new AssetType enum instead and use it only in AssetInfo?
Should we stick to the first approach and return mixed content `ArrayInfo | Group`? We can merge the assets properly in a future iteration.

Just my thoughts.

Reply:
Eventually we want to apply it universally. For now it will be only ONLY by groups. All other operations should continue to send empty file type when reffering to arrays